lib/client/ground.db.js

```diff
@@ -23,65 +22,7 @@ Ground = {};
 
 import localforage from 'localforage';
 import { ProgressCount } from './pending.jobs';
-import ServerTime from './servertime';
-
-// Adapted from underscore
-function throttle (func, wait, options) {
-  var timeout, context, args, result;
-  var previous = 0;
-  if (!options) options = {};
-
-  var later = function() {
-    previous = options.leading === false ? 0 : Date.now;
-    timeout = null;
-    result = func.apply(context, args);
-    if (!timeout) context = args = null;
-  };
-
-  var throttled = function() {
-    var now = Date.now;
-    if (!previous && options.leading === false) previous = now;
-    var remaining = wait - (now - previous);
-    context = this;
-    args = arguments;
-    if (remaining <= 0 || remaining > wait) {
-      if (timeout) {
-        clearTimeout(timeout);
-        timeout = null;
-      }
-      previous = now;
-      result = func.apply(context, args);
-      if (!timeout) context = args = null;
-    } else if (!timeout && options.trailing !== false) {
-      timeout = setTimeout(later, remaining);
-    }
-    return result;
-  };
-
-  throttled.cancel = function() {
-    clearTimeout(timeout);
-    previous = 0;
-    timeout = context = args = null;
-  };
-
-  return throttled;
-};
-
-// Adapted from you don't need underscore
-export const difference = (a, b) =>
-  [a, b].reduce((a, b) => a.filter(c => !b.includes(c)));
-
-// Without the Kernel
-if (typeof Kernel === 'undefined') {
-  var Kernel = {
-    defer(f) {
-      Meteor.setTimeout(f, 0);
-    },
-    each(items, f) {
-      items.forEach(f);
-    },
-  };
-}
```

Comment:
This Kernel check should stay, Kernel is used in loadDatabase and fails when 'undefined'.

Reply:
I reverted that. Should be good to go.